packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -163,7 +163,7 @@ export function getInputFieldTypeName(modelName: string, field: GraphQLField<any
 
     const relationshipAnnotation = parseRelationshipAnnotation(field.description);
 
-    if (!relationshipAnnotation) {
+    if (!relationshipAnnotation && relationshipAnnotation.kind !== 'manyToOne') {
```

Comment:
I think this will always fail with `cannot read property kind of undefined` since the relationship is always undefined by the `!relationshipAnnotation` check as seem to be indicated by CI: https://github.com/aerogear/graphback/pull/2020/checks?check_run_id=1090647585

This is too specific for `manyToOne` relationships and I am wondering what happens for other relationships?

Maybe https://github.com/aerogear/graphback/issues/2019#issuecomment-689514459 is what we need here /cc @craicoverflow and generally I think we need an issue that highlights relationships enhancement.

Reply:
It should have ? Added